tests/test-cases/Group23-VIC-Machine-Service/23-04-VCH-Inspect.robot

```diff
@@ -151,3 +165,97 @@ Get Invalid VCH
 
 Get Invalid VCH Within Datacenter
     Get Path Under Target  /datacenter/${DC-ID}/vch/INVALID
+    
+Get Deleted VCH
+    Create VCH    '{"name":"%{VCH-NAME}-api-test-simple","compute":{"resource":{"name":"%{TEST_RESOURCE}"}},"storage":{"image_stores":["ds://%{TEST_DATASTORE}"]},"network":{"bridge":{"ip_range":"172.16.0.0/12","port_group":{"name":"%{BRIDGE_NETWORK}"}},"public":{"port_group":{"name":"${PUBLIC_NETWORK}"}}},"auth":{"server":{"generate":{"cname":"vch.example.com","organization":["VMware, Inc."],"size":{"value":2048,"units":"bits"}}},"client":{"no_tls_verify": true}}}'
+    Verify Return Code
+    Verify Status Created
+          
+    ${expectedId}=  Get VCH ID  %{VCH-NAME}-api-test-simple
+    Get Path Under Target  vch/${expectedId}
+    Run Secret VIC Machine Delete Command    %{VCH-NAME}-api-test-simple
+            
+    Get Path Under Target  vch/${expectedId}
+    Verify Return Code
+    Verify Status Not Found
+    
+    Output Should Contain   unable to find VCH
+    
+Get Inspect for non-VCH VM
+    ${rc}  ${compute-path}=  Run And Return Rc And Output  govc ls host | grep %{TEST_RESOURCE}
+    Should Be Equal As Integers  ${rc}  0
+    # Create dummy VM
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.create -pool=${compute-path} -net=%{PUBLIC_NETWORK} Dummy_VM
+    Should Be Equal As Integers  ${rc}  0
+    
+    Get Path Under Target  vch/Dummy_VM
```

Comment:
It seems like this should use `Dummy_VM`'s ID (looking up by name won't work for a VCH either).